core/domain/classifier_services.py

```diff
@@ -417,12 +433,35 @@ def _add_examples(self, examples, iterations):
     def add_examples_for_training(self, training_examples):
         """Adds examples to the classifier with _training_iterations number of
         iterations.
+
+        Args:
+            training_examples: list of examples. A single example is a doc
+                with a list of labels it should be matched to, e.g:
+
+                training_examples = [
+                    ['i eat fish and vegetables', ['food']],
+                    ['fish are pets', ['pets']],
+                    ['my kitten eats fish', ['food', 'pets']]
+                ]
+
+        Returns:
+            xrange. An iterator over the ids of the docs just added.
         """
         return self._add_examples(training_examples, self._training_iterations)
 
     def add_examples_for_predicting(self, prediction_examples):
         """Adds examples to the classifier with _prediction_iterations number
         of iterations.
+
+        Args:
+            prediction_examples: list of str. A list of docs.
```

Comment:
I'll edit  [core/controllers/reader.py](https://github.com/oppia/oppia/blob/develop/core/controllers/reader.py#L100) accordinly also the StringClassifier class [docstring](https://github.com/oppia/oppia/blob/develop/core/domain/classifier_services.py#L45).